Fetchdatamodel

RuthDibenzi opened the pull request.

RuthDibenzi committed:
Added DailyQuizScore controller, entity, repository, service

RuthDibenzi committed:
risk score calculation

jmacap committed:
Bug fix: rest template needed to be specified, otherwise application cannot run

jmacap committed:
Form & Security Config: Changed cors configuration, + testing POSTs using Form.jsx

RuthDibenzi committed:
Daily form is now accessible, the endpoints are set up allowing post and get requests, updated pom.xml file, security config file allows dailyscore cors authorisation, loggers added to service class - dailyquizescore service to inspect risk calculation

RuthDibenzi committed:
added local host form

RuthDibenzi committed:
fetch data

RuthDibenzi committed:
changes

Commit:
model calling issue

Commit:
faga

Naveen-Jaisankar committed:
added post request for dailyformquiz

jmacap committed:
Yeehaw, dailyform can access the database
Daily form successfully sends information with mock ID and mock quizscore. Many lines that are print statements have been commented out for easier testing & potentially
will be removed in future updates depending on best maintainability practices.

jmacap committed:
UserDashboard: successful Rudimentary Get request, will modify so that when page loads it'll automatically send a get request & without reloading

jmacap committed:
Rudimentary restructure, works for POST requests but not GET requests

jmacap committed:
MAJOR UPDATE: BACKEND DATABASE NOW FUNCTIONS WITH PRIMARY COMPOSITE KEY. The following files have been changed: BACKEND - adjusted DailyQuizService, DailyQuizController to take in the ID entity as well as modified what counts as a primary key in the DailyQuizRepository. FRONTEND - modified the axios GET+ POST queries in both DailyForm + Form, and works beautifully(on my machine).

RuthDibenzi committed:
open weather api key placed in application.ym file, service class updated with risk assesment alorithim and access to the data model using rest api

RuthDibenzi committed:
Updated Java files related to DailyQuizScore and utilities

Naveen-Jaisankar commented:
Could you please remove node_module from this repo...Please add that to .gitignore...

RuthDibenzi committed:
pulled from main and fixed merge conflicts

RuthDibenzi committed:
remove node mdoules

Naveen-Jaisankar commented on Backend/src/main/java/com/compsci/webapp/config/SecurityConfig.java:
```diff
@@ -49,7 +51,7 @@ SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
     @Bean
     CorsConfigurationSource corsConfigurationSource() {
         CorsConfiguration configuration = new CorsConfiguration();
```
Any specific reason behind adding trailing slash?

RuthDibenzi replied:
ensures sub paths are covered, we don't have any at the moment but its good practice

Naveen-Jaisankar replied:
we'll have cors issue in the production with trailing slash that's why i removed it and added a comment there

Naveen-Jaisankar commented on DataAnalytics/model_flask_app/.idea/app.py:
We already have these files under data analytics folder....

RuthDibenzi replied:
I deleted them

Naveen-Jaisankar commented on FrontEnd/air-quality-app/src/App.jsx:
You have remove most of the routes that are recently pushed into main branch...

RuthDibenzi committed:
Delete DataAnalytics/model_flask_app/.idea/app.py

deleted duplicate files

RuthDibenzi reviewed (commented):
removed duplicate files

Ritwik-Ritz committed:
Register page image fix

Ritwik-Ritz committed:
Merge branch 'main' into frontend-react2

RuthDibenzi committed:
fixed app.js routes and added user dashboard file

RuthDibenzi committed:
Resolved conflicts and updated Backend and FrontEnd files

RuthDibenzi committed:
updated the coding standard changes in controller class, solved merge conflict

RuthDibenzi committed:
Update Java files in Backend for DailyQuizScore feature

RuthDibenzi committed:
corrected dependancies that were missing

RuthDibenzi committed:
import axios path changed as it could not
find the file which axios instance was defined, changed to src/axios as it was api/base which does not exist

RuthDibenzi committed:
Update package-lock.json

RuthDibenzi committed:
updated file added

RuthDibenzi committed:
removed duplicate data files

RuthDibenzi committed:
removed dummy file

RuthDibenzi committed:
Remove unused files from model_flask_app .idea directory

RuthDibenzi committed:
Merge remote-tracking branch 'origin/frontend-react2' into fetchdatamodel - getting updated changes

RuthDibenzi committed:
Update SecurityConfig.java removal of trail

RuthDibenzi committed:
Improve DailyQuizScoreService.java logic and performance

RuthDibenzi committed:
updated file

RuthDibenzi committed:
Update App.jsx to allow access without protection can change back

RuthDibenzi committed:
added import statement for side bar and added file path for axios instance

RuthDibenzi committed:
Remove obsolete App.jsx from static directory

RuthDibenzi committed:
modifiedaxios request to allow data reponse and sending
